stackslib/src/config/mod.rs

```diff
@@ -1996,134 +2008,151 @@ impl BurnchainConfigFile {
 pub struct NodeConfig {
     /// Human-readable name for the node. Primarily used for identification in testing environments
     /// (e.g., deriving log file names, temporary directory names).
-    ///
-    /// Default: `"helium-node"`
+    /// ---
+    /// @default: `"helium-node"`
     pub name: String,
     /// The node's Bitcoin wallet private key, provided as a hex string in the config file.
     /// Used to initialize the node's keychain for signing operations.
-    /// If `miner.mining_key` is not set, this seed may also be used for mining-related signing.
-    /// Required if [`NodeConfig::miner`] is `true` and [`MinerConfig::mining_key`] is absent.
-    ///
-    /// Default: Randomly generated 32 bytes.
+    /// If [`MinerConfig::mining_key`] is not set, this seed may also be used for mining-related signing.
+    /// ---
+    /// @default: Randomly generated 32 bytes
+    /// @notes:
+    ///   - Required if [`NodeConfig::miner`] is `true` and [`MinerConfig::mining_key`] is absent.
     pub seed: Vec<u8>,
     /// The file system absolute path to the node's working directory.
     /// All persistent data, including chainstate, burnchain databases, and potentially other stores,
     /// will be located within this directory.
     /// This path can be overridden by setting the `STACKS_WORKING_DIR` environment variable.
-    /// Note: For persistent mainnet or testnet nodes, this path must be explicitly
-    /// configured to a non-temporary location in the configuration file.
-    ///
-    /// Default: `/tmp/stacks-node-{current_timestamp}`.
+    /// ---
+    /// @default: `/tmp/stacks-node-{current_timestamp}`
+    /// @notes:
+    ///   - For persistent mainnet or testnet nodes, this path must be explicitly configured to
+    ///     a non-temporary location.
     pub working_dir: String,
     /// The IPv4 address and port (e.g., "0.0.0.0:20443") on which the node's HTTP RPC server
     /// should bind and listen for incoming API requests.
-    ///
-    /// Default: `"0.0.0.0:20443"`
+    /// ---
+    /// @default: `"0.0.0.0:20443"`
     pub rpc_bind: String,
     /// The IPv4 address and port (e.g., "0.0.0.0:20444") on which the node's P2P networking
     /// service should bind and listen for incoming connections from other peers.
-    ///
-    /// Default: `"0.0.0.0:20444"`
+    /// ---
+    /// @default: `"0.0.0.0:20444"`
     pub p2p_bind: String,
     /// The publicly accessible URL that this node advertises to peers during the P2P handshake
     /// as its HTTP RPC endpoint. Other nodes or services might use this URL to query the node's API.
-    ///
-    /// Default: `http://{rpc_bind}` (e.g., "http://0.0.0.0:20443" if rpc_bind is default).
+    /// ---
+    /// @default: `http://{rpc_bind}` (e.g., "http://0.0.0.0:20443" if [`NodeConfig::rpc_bind`] is default).
```

Comment:
would this explanation fit better as a `@note` (side effect of reducing the line length in the resultant docs)?
ex:
```
@notes 
  - if [`NodeConfig::rpc_bind`] is default: http://0.0.0.0:20443
```